php/simps/composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "require": {
-    "simple-swoole/simps": "dev-master"
+    "simple-swoole/simps": "^1.0"
```

Comment:
I prefer using `~`. Using carret will break logic, it could install **1.1**, but generate a `README` with a wrong information :stuck_out_tongue:

Reply:
only the case if using `~1.0.0`